internal/meta/meta_res.go

```diff
@@ -63,8 +67,9 @@ func (meta *MetaResource) ListResource(ctx context.Context) (ImportList, error)
 		resources = append(resources, resourceset.AzureResource{Id: id})
 	}
 
-	rset := &resourceset.AzureResourceSet{
-		Resources: resources,
+	rset, err := meta.queryResourceSet(ctx, resources)
```

Comment:
This is only necessary if `includeRoleAssignment = true`, otherwise, we don't need to call `azlist`, to save one or more APIs.

Reply:
Fixed